### PR TITLE
fixing username changes breaking some quotes

### PR DESCRIPTION
**Pull request by justanotherfailure**
fixing username changes breaking quotes

### CODE REVIEW

**Comment by joker314** [comment]
Note to reviewers: it appears the line endings have changed. This may not be desirable, and might need to be fixed, but in the mean time, select "Ignore whitespace" in the diff settings to get a more manageable diff.

**Comment by justanotherfailure** [comment]
> 
> 
> Note to reviewers: it appears the line endings have changed. This may not be desirable, and might need to be fixed, but in the mean time, select "Ignore whitespace" in the diff settings to get a more manageable diff.

This is due to me simply uploading and replacing the files on my local repo, had i used normal git commands this probably wouldn't have happened, apologies.

File: bot/cogs/fun.py [hunk 1]
```diff
@@ -1,257 +1,257 @@
...
+from itertools import cycle
+from random import choice, randint
+from string import ascii_lowercase
+from typing import AsyncGenerator
+from urllib.parse import urlencode
+
+from aiohttp import ClientSession
+from discord import Embed, File, Message
+from discord.ext.commands import (
+    Bot, Context, TextChannelConverter, command, has_any_role, UserConverter
```
Comment (joker314) [inline bot/cogs/fun.py]: We put the uppercase things first, then the lowercase things, and each is sorted alphabetically. In this case,
```suggestion
    Bot, Context, TextChannelConverter, UserConverter, command, has_any_role
```

**Comment by justanotherfailure** [comment]
It seems that I hadn't remembered to cache new quotes while the bot was running, I've added that now with my last 2 commits.

**Comment by justanotherfailure** [comment]
imma close this, and make a proper one without broken line endings

**Pull request closed without merging**